### ISSUE TITLE
音声合成関連

### ISSUE THREAD

**Issue by mahiro1010** [mahiro1010/reference-article#4]

**Comment by mahiro1010** [comment]
* [リスニングとリコール時の脳波からの母音合成](https://onlinelibrary.wiley.com/doi/10.1002/aisy.202000164)
* [ヒトからコウモリに至る超聴覚：超低速高効率知覚機構](http://journal.vrsj.org/14-2/s17-22.pdf)
* [脳の聴覚情報処理](file:///Users/matsuurahirokazu/Downloads/IPSJ-MGN300205.pdf)
* [音を聴 く聴覚の 仕組み](https://www.jstage.jst.go.jp/article/jasj/66/9/66_KJ00006579569/_pdf)
* [音色とラウドネスの脳処理: 基本的な聴覚弁別に対するブローカ野の寄与に関する fMRI の証拠](https://link.springer.com/article/10.1007/s11682-007-9010-3)
* [スペクトル ラウドネスの合計は、一次聴覚皮質で行われます](https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.21123?casa_token=f0ZqHbVpXc4AAAAA%3AEuknygkEEqGCHXE3fYARCuLO6inXeC85bPnkswdnzYONwQHnrf0F4BAGYiV-zkoIUL3eytDYrJlTH16xFg)
* [2 章 聴覚と音声](http://www.ieice-hbkb.org/files/S3/S3gun_02hen_02.pdf)
* [末梢耳モデルに基づく発声包絡線の聴覚性中脳における神経表現](https://jglobal.jst.go.jp/detail?JGLOBAL_ID=201402219760475770)

**Comment by mahiro1010** [comment]
# 読みたい論文

## EEG, ECoG

* [Speech Synthesis Using EEG](https://ieeexplore.ieee.org/abstract/document/9053340)
* [Decoding speech from spike-based neural population recordings in secondary auditory cortex of non-human primates](https://www.nature.com/articles/s42003-019-0707-9)
* [Real-time decoding of question-and-answer speech dialogue using human cortical activity](https://www.nature.com/articles/S41467-019-10994-4)
* [Transformer-Based Estimation of Spoken Sentences Using Electrocorticography](https://ieeexplore.ieee.org/abstract/document/9747443)
* [VStyclone: Real-time Chinese voice style clone](https://www.sciencedirect.com/science/article/pii/S0045790622007492)
* [Data-driven models in human neuroscience and neuroengineering](https://www.sciencedirect.com/science/article/pii/S0959438818302502)
* [A Survey on Neural Speech Synthesis](https://arxiv.org/abs/2106.15561)
* [Speech synthesis from neural decoding of spoken sentences](https://www.nature.com/articles/s41586-019-1119-1.)

## Diffusion Model

* [DiffWave: A Versatile Diffusion Model for Audio Synthesis](https://arxiv.org/abs/2009.09761)
* [Diff-TTS: A Denoising Diffusion Model for Text-to-Speech](https://arxiv.org/abs/2104.01409)

**Comment by mahiro1010** [comment]
# メモ

## Transfer Learning(転移学習)
→ 個人間の違いを吸収するようなモデルの開発が可能になる？

### 

## Data Augmentation(DA)
→ 少量データによる生体信号のDL学習モデルの限界を克服できる？

**Comment by mahiro1010** [comment]
# わかりやすい資料

## MLP-Mixer
* https://cometscome.github.io/DLAP2020/slides/MLPMixer.pdf

**Comment by mahiro1010** [comment]
# 学習中キーワード

差分エントロピー(differential entropy (DE))
幾何学的変換とノイズ付加
シフト、スケール、回転/反射
ガウス、ポアソン、ソルト、ペッパー
強いランダム性と非定常性
Poisson noise, Salt noise, or Pepper noise, which will change the features of EEG data locally.
SVM)、LeNet、ResNet
セッション/セグメント
Batch normalization→学習の高速化
Dropout→学習の高速化
degradation
Heの初期値(ReLU)/Xavier(Sigmoid/tanh)
ReLu
tanh
sigmoid
恒等写像
GANの基礎
Ground truth
ミニバッチ
白色化=無相関化+標準化
標準化(Normalization)は「平均を0，分散を1とするスケーリング手法」
正規化(Standadization, Min-Max)は「最小値を0，最大値を1とする0-1スケーリング手法」
Internal Covariate Shift

**Comment by mahiro1010** [comment]
# WGANに関する参考記事

* [WGAN-GP（Train Wasserstein GAN with Gradient Penalty）](https://taviko.com/gan/wgan-gp/)
* [Wasserstein GAN（WGAN）でいらすとや画像を生成してみる](https://yusuke-ujitoko.hatenablog.com/entry/2017/05/20/145924)
* [PytorchでGANを実装してみた。](https://qiita.com/keiji_dl/items/45a5775a361151f9189d)
* [GAN (Generative Adversarial Network) の理論と実装](https://www.bigdata-navi.com/aidrops/2519/)
* [TensorFlow の使い方練習２：Functional API](https://zenn.dev/wsuzume/articles/cb1511666e2f99)
* [マルチモーダル・ディープラーニングの商品分類を本番環境で動かす](https://engineering.mercari.com/blog/entry/2019-02-19-180000/)
* [「From GAN to WGAN」 を和訳しながら理解したい](https://note.com/29081998/n/n67c9a0b9ae37)
* [From GAN to WGAN](https://lilianweng.github.io/posts/2017-08-20-gan/)
* [DCGAN-tensorflow](https://github.com/carpedm20/DCGAN-tensorflow)
* [Pythonによるデータ処理5 ～ フィルタ処理2](https://atatat.hatenablog.com/entry/data_proc_python5)
* [PythonのSciPyでローパスフィルタをかける！](https://watlab-blog.com/2019/04/30/scipy-lowpass/)
